core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1193,6 +1193,15 @@ package object config {
       .intConf
       .createWithDefault(3)
 
+  private[spark] val SHUFFLE_MAX_ATTEMPTS_ON_NETTY_OOM =
+    ConfigBuilder("spark.shuffle.nettyOOM.maxAttempts")
```

Comment:
This is introducing a new namespace `nettyOOM` but we have only one configuration for now. Do we have a plan to add more configuration under this namespace?

Reply:
For the conf namespace naming advice, cc @cloud-fan .

Reply:
renamed to `spark.shuffle.maxAttemptsOnNettyOOM`